### PR TITLE
lightning: convert pebbleDB block size to default when user passes in 0

### PR DESCRIPTION
**Pull request by lyzx2001**
### What problem does this PR solve?

Issue Number: ref #45037 

Problem Summary:

### What changed and how does it work?
convert pebbleDB block size to default when user passes in 0
### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No need to test
  > - [ ] I checked and no code files have been changed.
  > 

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by lance6716 — APPROVED** [review]

File: br/pkg/lightning/config/config.go [hunk 1]
```diff
@@ -100,6 +100,7 @@ const (
 
 	DefaultEngineMemCacheSize      = 512 * units.MiB
 	DefaultLocalWriterMemCacheSize = 128 * units.MiB
+	DefaultBlockSize               = 16 * units.KiB
```
Comment (D3Hunter) [inline br/pkg/lightning/config/config.go:103]: can you update this setting for `import into`
https://github.com/pingcap/tidb/blob/cf63fddd9b8491af6374e9eac36ca48f5640ba36/pkg/executor/importer/import.go#L1318-L1319

Comment (lyzx2001) [reply br/pkg/lightning/config/config.go:103]: modified

**Review by D3Hunter — APPROVED** [review]

**Pull request merged**